src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block">
```

Comment:
The 'block' class for the parent div seems unnecessary. Consider removing it or giving it a more descriptive name like 'stars-container'.